extensions/tika/runtime/src/main/java/io/quarkus/tika/runtime/TikaConfiguration.java

```diff
@@ -26,23 +27,44 @@
      * property is recommended to achieve both optimizations.
      * <p>
      * Either the abbreviated or full parser class names can be used.
-     * At the moment only PDF parser can be listed using a reserved 'pdf' abbreviation.
+     * Only PDF and OpenDocument format parsers can be listed using the reserved 'pdf' and 'odf' abbreviations.
      * Custom class name abbreviations have to be used for all other parsers.
      * For example:
      * 
      * <pre>
      * // Only PDF parser is required:
-     * tika-parsers = pdf
-     * // Only PDF and Java class parsers are required:
-     * tika-parsers = pdf,classparser
-     * classparser = org.apache.tika.parser.asm.ClassParser
+     * quarkus.tika.parsers = pdf
+     * // Only PDF and OpenDocument parsers are required:
+     * quarkus.tika.parsers = pdf,odf
      * </pre>
      * 
      * This property will have no effect if the `tikaConfigPath' property has been set.
      */
     @ConfigItem
     public Optional<String> parsers;
 
+    /**
+     * Configuration of the individual parsers.
+     * For example:
+     *
+     * <pre>
+     * quarkus.tika.parsers = pdf,odf
+     * quarkus.tika.parser.pdf.sort-by-position = true
+     */
+    @ConfigItem
+    public Map<String, Map<String, String>> parser;
+
+    /**
+     * Full parser class name for a given parser abbreviation.
+     * For example:
+     *
+     * <pre>
+     * quarkus.tika.parsers = classparser
+     * quarkus.tika.parser-name.classparser = org.apache.tika.parser.asm.ClassParser
+     */
+    @ConfigItem
+    public Map<String, String> parserName;
```

Comment:
I would have named this one `parser` and the other one `parserOptions`.

That would be far more logical.

Reply:
@gsmet, yes, I updated the code, makes perfect sense, thanks, however it looks like there is a minor regression on the config path...